scripts/superdesk-search/react.js

```diff
@@ -1576,6 +1588,35 @@ import classNames from 'classnames';
                             }, 500, false);
                         },
 
+                        selectItem: function (item) {
+                            if (isCheckAllowed(item)) {
+                                var selected = !item.selected;
+                                this.multiSelect([item], selected);
+                            }
+                        },
+
+                        selectMultipleItems: function (lastItem) {
+                            var itemId = search.generateTrackByIdentifier(lastItem),
+                                positionStart = 0,
+                                positionEnd = _.indexOf(this.state.itemsList, itemId),
+                                selectedItems = [];
+
+                            if (this.state.selected) {
+                                positionStart = _.indexOf(this.state.itemsList, this.state.selected);
+                            }
+
+                            for (var i = positionStart; i <= positionEnd; i++) {
+                                var item = this.state.itemsById[this.state.itemsList[i]];
+                                if (isCheckAllowed(item)) {
+                                    selectedItems.push(item);
+                                }
+                            }
+
+                            multi.reset();
```

Comment:
If user selects the first item by single select then selects items 3,4,5 with multiselect, will this line clear the previously selected first item?